lib/MusicBrainz/Server.pm

```diff
@@ -28,8 +28,6 @@ Session::State::Cookie
 
 Cache
 Authentication
-
-Unicode::Encoding
```

Comment:
No longer necessary since Catalyst 5.90040 (see http://wiki.catalystframework.org/wiki/tutorialsandhowtos/using_unicode.view). Only function now is to emit a deprecation warning.